spec/policy/find_service_spec.lua

```diff
@@ -0,0 +1,108 @@
+local configuration = require('configuration')
+
+describe('find_service', function()
+  describe('.rewrite', function()
+    describe('when path routing is enabled', function()
```

Comment:
I realized that I didn't test the case where no mapping rules are matched and find_service falls back to 'strict' mode.

Reply:
Done.